readme.md

```diff
@@ -27,11 +27,11 @@ This will allow Cargo to download, build, and cache Windows support as a package
 ```rust
 fn main() {
   windows::build!(
-      Windows::Win32::WindowsProgramming::CloseHandle,
-      Windows::Win32::WindowsAndMessaging::MessageBoxA,
-      Windows::Data::Xml::Dom::*,
-      Windows::Win32::SystemServices::{
-          CreateEventW, SetEvent, WaitForSingleObject
+      windows::win32::windows_programming::CloseHandle,
+      windows::win32::windows_and_messaging::{MessageBoxA, HWND, MB_FLAGS},
+      windows::data::xml::dom::*,
+      windows::win32::system_services::{
+          CreateEventW, SetEvent, WaitForSingleObject, PWSTR
```

Comment:
This is undoing the changes needed as a result of [this pull request](https://github.com/microsoft/windows-rs/pull/646).

Reply:
The current README examples don't work with the latest version on cargo.

Reply:
Correct. But they do work with the latest version on the *master* branch. This is what's going to go into the next revision, and the README has been updated to reflect that already. That's the reason why all the tests are failing.

Reply:
Any timeline for a new crate release?

Reply:
Usually once a week. Probably on Monday.